android-sdk/tasks/main.yml

```diff
@@ -165,3 +170,43 @@
   when: "'Accept?' in rsh_cmd.stdout"
   tags:
     - install-sdk
+
+-
+  name: "Check gradle wrapper exists"
+  command: oc exec --namespace={{ project_name }} {{ android_sdk_podname }} -i -- bash -c 'stat $ANDROID_HOME/tools/templates/gradle/wrapper/gradle-wrapper.tar.gz'
```

Comment:
I think we have to change "$ANDROID_HOME" to "{{ android_sdk_home }}" as it is using the env var from the host using ansible rather than the one defined in the container/pod.